recipe/common.php

```diff
@@ -273,27 +274,36 @@
     $sudo = get('writable_use_sudo') ? 'sudo' : '';
     $httpUser = get('http_user');
 
+    $users = env('writable_users');
+    $permissions = '';
+
     if (!empty($dirs)) {
         try {
             if (null === $httpUser) {
                 $httpUser = run("ps aux | grep -E '[a]pache|[h]ttpd|[_]www|[w]ww-data|[n]ginx' | grep -v root | head -1 | cut -d\  -f1")->toString();
             }
 
+            $users[] = $httpUser;
```

Comment:
<pre><code>if(!in_array($httpUser, $users)){
    $users[] = $httpUser;
}</code></pre>

The same for <code>`whoami`</code>. But, i move forward with this or close this and wait for your solution as you said in #612?